polaris-ui/src/app/common/presentation/components/Modal.module.scss

```diff
@@ -10,8 +10,9 @@
   //  the wheel and starts scrolling itself, rather than the modal content.
   // 3) z-index: 3 seems to be given to the text and annotation layer of pdf.js
   // 4) z-index: 4 is used by the pdf viewer redaction summary footer
-  // So use z-index: 5
-  z-index: 5;
+  // 4) z-index: 6 is used by the redaction button
```

Comment:
Change the bullet point number to 5).  Although the bullet point numbering can confuse the reader as it does not keep in step with the z-index numbering, so maybe just remove the bullet point numbering altogether.  Will leave it up to you.

Reply:
fixed